blocks/tabs/tabs.js

```diff
@@ -7,51 +12,62 @@ import {
   button,
 } from '../../scripts/dom-helpers.js';
 
-export default function decorate(block) {
-  document.querySelector('.tabs-container').classList.add('calcite-mode-dark');
+export default async function decorate(block) {
+  const isTabsCardsVariant = block.classList.contains('tabs-cards-variant');
 
   block.querySelectorAll('img').forEach((img) => img
     .closest('picture')
     .replaceWith(
       createOptimizedPicture(img.src, img.alt, false, [{ width: '750' }]),
     ));
 
-  const tabTitles = [...block.children].map((child) => child.children[0].children[0].textContent);
-  const tabContents = [...block.children].map((child) => [...child.children[1].children]);
-
-  tabContents.forEach((content) => {
-    const text = [content[1], content[2], content[3]];
-    const textWrapper = div({ class: 'text-wrapper' }, ...text);
-    content.splice(1, 3, textWrapper);
-
-    const hrefs = [content[2].children[0].href, content[3].children[0].href];
-    const buttons = [
-      calciteButton({
-        'icon-end': 'play-f',
-        href: hrefs[0],
-        appearance: 'solid',
-        alignment: 'center',
-        scale: 'm',
-        type: 'button',
-        width: 'auto',
-        kind: 'inverse',
-        color: 'inverse',
-      }, content[2].textContent),
-      calciteButton({
-        'icon-end': 'arrowRight',
-        href: hrefs[1],
-        appearance: 'outline',
-        alignment: 'center',
-        scale: 'm',
-        type: 'button',
-        width: 'auto',
-        kind: 'inverse',
-      }, content[3].textContent),
-    ];
-
-    const buttonsWrapper = div({ class: 'buttons-wrapper' }, ...buttons);
-    content.splice(2, 2, buttonsWrapper);
-  });
+  const loadBlocks = [];
+  let tabTitles;
+  let tabContents;
+  if (!isTabsCardsVariant) {
+    document.querySelector('.tabs-container').classList.add('calcite-mode-dark');
+
+    tabTitles = [...block.children].map((child) => child.children[0].children[0].textContent);
+    tabContents = [...block.children].map((child) => [...child.children[1].children]);
+
+    tabContents.forEach((content) => {
+      const text = [content[1], content[2], content[3]];
+      const textWrapper = div({ class: 'text-wrapper' }, ...text);
+      content.splice(1, 3, textWrapper);
+
+      const hrefs = [content[2].children[0].href, content[3].children[0].href];
+      const buttons = [
+        calciteButton({
+          'icon-end': 'play-f',
+          href: hrefs[0],
+          appearance: 'solid',
+          alignment: 'center',
+          scale: 'm',
+          type: 'button',
+          width: 'auto',
+          kind: 'inverse',
+          color: 'inverse',
+        }, content[2].textContent),
```

Comment:
you're duplicating content[2] and content[3], can you move them to a variable for readability?